binance/websockets.py

```diff
@@ -11,6 +11,16 @@
 from twisted.internet.error import ReactorAlreadyRunning
 
 from binance.client import Client
+import logging
+
+logger = logging.getLogger('binance.websockets')
+
+
+class RepeatTimer(threading.Timer):
+    """Source: https://stackoverflow.com/a/48741004/304209."""
+    def run(self):
+        while not self.finished.wait(self.interval):
+            self.function(*self.args, **self.kwargs)
```

Comment:
it should be:
```
    def run(self):
        while not self.finished.wait(self.interval):
            if not self.finished.is_set():
                self.function(*self.args, **self.kwargs)
```
Otherwise if someone call `cancel()` it would run `self.function()` careless

Reply:
Done, thanks!